packages/manager/modules/vps/src/upscale/summary-price/summary-price.html

```diff
@@ -0,0 +1,29 @@
+<span class="oui-box d-block py-2 border-0 rounded-0">
+    <span class="d-block text-right vps-upscale-summary-price__price-frequency">
+        <span
+            data-ng-if="$ctrl.paymentType.value === 'monthly'"
```

Comment:
payment type values can be constants?

Reply:
or at least use a method to avoid exposing constants. @radireddy there are defined here : packages/manager/modules/vps/src/upscale/summary-price/summary-price.constants.js

Reply:
Yup good idea